airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -134,6 +134,11 @@ definitions:
         type: array
         items:
           "$ref": "#/definitions/SyncMode"
+      cursor_field:
```

Comment:
we could replace this with a boolean. with a source-defined cursors, it's not technically necessary to expose them outside the source for the actual replication. i exposed it in this iteration so that the UI would be able to display what field would be used (even if it's not configurable). i can go either way on this one.

Reply:
cool. going with the boolean as you suggest. called `source_defined_cursor_field`